Scripts/InstallFEX.py

```diff
@@ -306,19 +306,19 @@ def GetRootFSPath():
     return _RootFSPath
 
 def CheckRootFSInstallStatus():
-    # Matches what is available on https://rootfs.fex-emu.gg/RootFS_links.json
-    UbuntuVersionToRootFS = {
-        "20.04": "Ubuntu_20_04.sqsh",
-        "20.04": "Ubuntu_20_04.ero",
-        "22.04": "Ubuntu_22_04.sqsh",
-        "22.04": "Ubuntu_22_04.ero",
-        "22.10": "Ubuntu_22_10.sqsh",
-        "22.10": "Ubuntu_22_10.ero",
-        "23.04": "Ubuntu_23_04.sqsh",
-        "23.04": "Ubuntu_23_04.ero",
-    }
-
-    return os.path.exists(GetRootFSPath() + UbuntuVersionToRootFS[GetDistro()[1]])
+    Distro = GetDistro()[1]
```

Comment:
Adding a small example here makes it easier to follow the logic below:

```suggestion
    Distro = GetDistro()[1] # Extract Ubuntu version number, e.g. "23.10"
```